Resources/config/utils.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="sonata.doctrine.utils.dump_directory">%kernel.root_dir%/cache/sonata/ecommerce</parameter>
```

Comment:
If you need a parameter, you most of the time rather need a configuration option or a function/command line parameter. Re-think this each time you face it.